src/main.js

```diff
@@ -107,17 +130,41 @@ export class Draggable {
         this._mouseup = (e) => {
             unbind(this.document, "mousemove", this._mousemove);
             unbind(this.document, "mouseup", this._mouseup);
+
+            if (this.clickMoveClick){
+                unbind(this.document, "keydown", this._keydown);
+                unbind(this.document, "contextmenu", preventDefault);
+            }
```

Comment:
Do we need to unbind other events, e.g.:

https://github.com/telerik/kendo-draggable/pull/53/files#diff-27653c212e1cfe533e4eb2f7d0d3f89604c9de48a09583b4cbbbcbd08a07da79R152